aos_head_generator/main.py

```diff
@@ -16,10 +16,13 @@ def create_head(color):
 
 @app.route("/v1/heads/<string>.png")
 def send_image(string):
-    if len(string) != 6:
+    if len(string) not in (6, 3):
         abort(400)
 
-    color_str = string[0:2], string[2:4], string[4:6]
+    l = len(str)//3
+    color_str = string[0:l], string[l:l*2], string[l*2:l*3]
+    color_str = map(lambda x: if len(x) < 2: x+x, color_str)
```

Comment:
This is invalid syntax. This line also seems unnecessary? `int(x, 16)` works with both 1 and 2 length strings. List comprehensions are also preferable to `map(lambda x: ...)` as they are a lot easier to read.

I'd suggest testing your code locally before submitting a PR.

Reply:
It is necessary as otherwise it would only set lower bytes for each color.